features/family/set_the_attribute_used_as_image.feature

```diff
@@ -0,0 +1,50 @@
+@javascript
+Feature: Set the attribute used as image
+  In order to let the user which attribute is the most accurate as the product image
+  As an administrator
+  I need to be able to set the attribute used as the image
+
+  Background:
+    Given the "default" catalog configuration
+    And the following attributes:
+      | label-en_US | type              | group | code  |
+      | Brand       | pim_catalog_text  | other | brand |
+      | Image       | pim_catalog_image | other | image |
+    And the following family:
+      | code | attributes |
+      | Bags | brand      |
+    And I am logged in as "Peter"
+
+  Scenario: Successfully show default attribute_as_image
+    Given I am on the "Bags" family page
+    Then I should see the text "No attribute set"
```

Comment:
No because the select2 have "no attribute set" value but displays "undefined" sometimes (known bug). I will let it as it to prevent this.